README.md

```diff
@@ -191,22 +191,41 @@ py.test
 #### The test data subset
 If you add new tables to the data, you'll need to generate a new subset for testing. We use this nifty subsetting tool: [rdbms-subsetter](https://github.com/18F/rdbms-subsetter).
 
-To build a new test subset, use the `build_test` invoke task:
+To build a new test subset, first create a new database for yourself:
+
+```
+createdb cfdm_new_subset
```

Comment:
Ah okay, you're referring to just using the existing `cfdm_test` database that we start out with when setting things up locally, right?  In that case you still need to drop the database and start again, but I can update the instructions to reflect that instead of just creating an entirely new database.